### PR TITLE
[Doc] Fix some incorrect uses of `a/an`

### PR DESCRIPTION
**Pull request by AThousandShips**
Treating "LOD" as being pronounced "eLODee", otherwise this is largely fixing cases spelled correctly elsewhere

### CODE REVIEW

File: doc/classes/RenderingDevice.xml [hunk 1]
```diff
@@ -636,7 +636,7 @@
 			<param index="0" name="shader_source" type="RDShaderSource" />
 			<param index="1" name="allow_cache" type="bool" default="true" />
 			<description>
-				Compiles a SPIR-V from the shader source code in [param shader_source] and returns the SPIR-V as a [RDShaderSPIRV]. This intermediate language shader is portable across different GPU models and driver versions, but cannot be run directly by GPUs until compiled into a binary shader using [method shader_compile_binary_from_spirv].
+				Compiles an SPIR-V from the shader source code in [param shader_source] and returns the SPIR-V as a [RDShaderSPIRV]. This intermediate language shader is portable across different GPU models and driver versions, but cannot be run directly by GPUs until compiled into a binary shader using [method shader_compile_binary_from_spirv].
```
Comment (akien-mga) [inline doc/classes/RenderingDevice.xml:639]: I think it's pronounced "spurr - vee" and not spelled out letter by letter, hence using `a`. That's debatable I guess.

Comment (AThousandShips) [reply doc/classes/RenderingDevice.xml:639]: Oh, that's odd, will restore

Comment (akien-mga) [reply doc/classes/RenderingDevice.xml:639]: Some "evidence":
http://casual-effects.blogspot.com/2016/03/computational-graphics-pronunciation.html
https://twitter.com/lunasorcery/status/1100506456732942336

File: doc/classes/SurfaceTool.xml [hunk 2]
```diff
@@ -124,7 +124,7 @@
 			<param index="0" name="nd_threshold" type="float" />
 			<param index="1" name="target_index_count" type="int" default="3" />
 			<description>
-				Generates a LOD for a given [param nd_threshold] in linear units (square root of quadric error metric), using at most [param target_index_count] indices.
+				Generates an LOD for a given [param nd_threshold] in linear units (square root of quadric error metric), using at most [param target_index_count] indices.
```
Comment (akien-mga) [inline doc/classes/SurfaceTool.xml:127]: I would say "lod" as a word, but I guess "el oh dee" is a valid form too.
Only "reference" I could find from a quick search is this shady looking https://www.spellchecker.net/lod

**Review by akien-mga — APPROVED** [review]

File: doc/classes/SurfaceTool.xml (hunk 2 above)
Comment (akien-mga) [reply doc/classes/SurfaceTool.xml]: Well I'm not sure on that one, might be worth polling on #rendering to see what folks are used to hearing :)

Comment (AThousandShips) [reply doc/classes/SurfaceTool.xml:127]: Given the current trend for the poll I added back the "LOD" cases, will squash or drop depending

Comment (akien-mga) [reply doc/classes/SurfaceTool.xml:127]: I think you can go ahead and squash :)

File: doc/classes/@GlobalScope.xml [hunk 3]
```diff
@@ -1099,14 +1099,14 @@
 		<method name="rid_allocate_id">
 			<return type="int" />
 			<description>
-				Allocates a unique ID which can be used by the implementation to construct a RID. This is used mainly from native extensions to implement servers.
```
Comment (Mickeon) [inline doc/classes/@GlobalScope.xml:1102]: This does not feel right, can you elaborate on what I missed?

I would replace all of these RID over the place with a direct reference to the **RID**, but that's for later.

It feels so wrong to me I came back to this message just to try to wrap my head around it. To me, RID reads as it is written. And the acronym stands for "Resource Identifier", not "Identifier Resource".

Comment (AThousandShips) [reply doc/classes/@GlobalScope.xml:1102]: It's "ar I dee", not "rid", this isn't about anything else than grammar

**Comment by akien-mga** [comment]
Thanks!

**Comment by AThousandShips** [comment]
Thank *you*!

**Pull request merged**